client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -3,34 +3,65 @@
  */
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
+import { reduxForm } from 'redux-form';
 import { localize } from 'i18n-calypso';
 import { flowRight, noop } from 'lodash';
 
 /**
  * Internal dependencies
  */
+import CompactCard from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
 import HeaderCake from 'components/header-cake';
+import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SectionHeader from 'components/section-header';
 import { getSelectedSiteSlug } from 'state/ui/selectors';
 
+const form = 'extensions.zoninator.newZone';
+
 const ZoneCreator = ( { siteSlug, translate } ) => (
 	<div>
 		<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` } onClick={ noop }>
 			{ translate( 'Add a zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Add a DNS Zone' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/zoninator-new-zone-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).